Common/sources/commondefines.js

```diff
@@ -852,7 +908,9 @@ const c_oPublishType = {
   meta: 11,
   forceSave: 12,
   closeConnection: 13,
-  changesNotify: 14
+  changesNotify: 14,
+  changeConnecitonInfo: 15,

```

Comment:
typo (this appears to be from the tag, and not you)

Reply:
This is from the tag